src/tests/App.test.js

```diff
@@ -1,5 +1,5 @@
 import { render, screen } from '@testing-library/react';
-import App from './App';
+import App from '../App';
```

Comment:
I think this might need to change to `'../pages/App'` given the new file structure you introduced